main.js

```diff
@@ -10,4 +11,288 @@ function closeMenu() {
+
+    liveLink: 'https://bilalrajput09.github.io/My-Portfolio/',
+    linkToSource: 'https://github.com/bilalrajput09',
+    technologies: ['HTML', 'CSS', 'Ruby On Rails'],
+  },
+  {
+    id: 4,
+    fullName: 'Project 4',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec auctor, nisl eget ultricies lacinia, nunc nisl aliquam nisl, eget aliquam nunc nisl eget nunc. Donec auctor, nisl eget ultricies lacinia, nunc nisl aliquam nisl, eget aliquam nunc nisl eget nunc.',
+
+    liveLink: 'https://bilalrajput09.github.io/My-Portfolio/',
+    linkToSource: 'https://github.com/bilalrajput09',
+    technologies: ['HTML', 'CSS', 'Ruby On Rails'],
+  },
+  {
+    id: 5,
+    fullName: 'Project 5',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec auctor, nisl eget ultricies lacinia, nunc nisl aliquam nisl, eget aliquam nunc nisl eget nunc. Donec auctor, nisl eget ultricies lacinia, nunc nisl aliquam nisl, eget aliquam nunc nisl eget nunc.',
+
+    liveLink: 'https://bilalrajput09.github.io/My-Portfolio/',
+    linkToSource: 'https://github.com/bilalrajput09',
+    technologies: ['HTML', 'CSS', 'Ruby On Rails'],
+  },
+  {
+    id: 6,
+    fullName: 'Project 6',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec auctor, nisl eget ultricies lacinia, nunc nisl aliquam nisl, eget aliquam nunc nisl eget nunc. Donec auctor, nisl eget ultricies lacinia, nunc nisl aliquam nisl, eget aliquam nunc nisl eget nunc.',
+
+    liveLink: 'https://bilalrajput09.github.io/My-Portfolio/',
+    linkToSource: 'https://github.com/bilalrajput09',
+    technologies: ['HTML', 'CSS', 'Ruby On Rails'],
+  },
+];
+
+function createProjectCards(projects) {
+  projects.forEach((project) => {
+    const projectCard = document.createElement('div');
+    projectCard.classList.add('cards');
+    projectCard.innerHTML = `
+              <div class="cards">
+            <h2 class="data">
+              ${project.fullName}
+            </h2>
+            <p class="para">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard.
+              And this si what has not been done daily.
+              has been the industry's standard.
+              And this si what has not been done daily.
+            </p>
+            <ul class="languages-2">
+              ${project.technologies.map((tech) => `<li class "card-tag" >${tech}</li>`).join('')}
+            </ul>
+            <a href="#" class="cards-project" data-target = "modal-${project.id}" onclick = "showModal(this)">See Project</a>
+          </div>
+          `;
+    works.appendChild(projectCard);
+  });
+}
```

Comment:
Well done on this project popup. 

- [ ] When the "See project" button is clicked, it appears very well on the Mozilla firefox browser but does not appear complete in the chrome browser. See below. My suggestion is that you add a scroll property to the container of the popup to fix this. 

Chrome Browser
--- |
![image](https://user-images.githubusercontent.com/84156714/223844209-3ff5fd8d-6203-412d-ba88-e2f38498846b.png)
The buttons in the popup above are hidden. If you add scrolling to the container, users can scroll down to see the button however it appears in any browser.

Firefox Browser
![image](https://user-images.githubusercontent.com/84156714/223844308-eeb942cc-849f-4a6f-b986-9a63270e842d.png)